dataikuapi/dss/ml.py

```diff
@@ -829,17 +829,28 @@ def set_range(self, min=None, max=None, nb_values=None):
 
     @property
     def range(self):
-        return Range(self)
+        return RangeSettings(self)
 
 
 class Range(object):
 
```

Comment:
Add a docstring here and for `RangeSettings` to be sure the user understands the difference ?